code/modules/mob/living/simple_animal/slime/life.dm

```diff
@@ -241,7 +241,7 @@
 		update_action_buttons_icon()
 
 	if(amount_grown >= SLIME_EVOLUTION_THRESHOLD && !buckled && !Target && !ckey)
-		if(is_adult)
+		if(is_adult && loc.AllowDrop())
```

Comment:
is this check needed since you updated the reproduce proc?

Reply:
Reproduce can be called by slime with mind.
This one for clientless slimes.
I place it here because hate slimes splitting out of slime console. As i see 2 slimes disappear on it.

First i think make slime pick up put slimes in stasis but this more complex and unnecessary.

Reply:
yes, even if it's functional code-wise, it pretty much never makes sense to make a slime teleport out of a container or console when splitting